api/v1alpha1/volumereplication_types.go

```diff
@@ -80,6 +81,11 @@ type VolumeReplicationStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:JSONPath=".metadata.creationTimestamp",name=Age,type=date
+// +kubebuilder:printcolumn:JSONPath=".spec.replicationState",name=desiredState,type=string
+// +kubebuilder:printcolumn:JSONPath=".status.state",name=currentState,type=string
+// +kubebuilder:printcolumn:JSONPath=".status.message",name=message,type=string
```

Comment:
Drop the message, it might not be helpful. If the users see that desired state and the current state are not matching they can always do describe and check the message.